### PR TITLE
Add CI/CD 

### PR DESCRIPTION
**Pull request by damir-abdulin**

### CODE REVIEW

**Review by emmanuel-montoya — APPROVED** [review]
awesome

File: .github/workflows/.example.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: Example CI/CD
```
Comment (KirillSergeevich) [inline .github/workflows/.example.yml:1]: it's weird name, isn't it?

File: .github/workflows/.example.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+          channel: 'stable'
+
+      - name: Install lcov
+        run: sudo apt-get install lcov
+
+      - name: Analyzing
+        run: flutter analyze
+
+      - name: Running test
+        run: flutter test --coverage
```
Comment (KirillSergeevich) [inline .github/workflows/.example.yml:36]: do we have tests for example to check its coverage?

File: .github/workflows/.example.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
+name: Example CI/CD
+
+on:
+  push:
+    branches: [ "master" ]
+    paths:
+      - 'example/**'
+  pull_request:
+    branches: [ "master" ]
```
Comment (KirillSergeevich) [inline .github/workflows/.example.yml:9]: did you check these pipelines? I am not sure that it works

File: .github/workflows/.example.yml (hunk 2 above)
Comment (damir-abdulin) [reply .github/workflows/.example.yml:36]: No, we haven't. Have to I write them?

File: .github/workflows/package.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+          channel: 'stable'
+
+      - name: Install lcov
+        run: sudo apt-get install lcov
+
+      - name: Analyzing
+        run: flutter analyze
+
+      - name: Running test
+        run: flutter test --coverage
```
Comment (KirillSergeevich) [inline .github/workflows/package.yml:24]: Do we have tests to check coverage?/?

File: .github/workflows/package.yml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+      - name: Coverage check
+        run: |
+          lcov --remove coverage/lcov.info '*.g.dart' '*.part.dart' '*/generated/*' -o coverage/lcov.info
+          ./scripts/pana_check.sh
```
Comment (KirillSergeevich) [inline .github/workflows/package.yml:29]: I am not sure it's a good idea to have pana check in step `Coverage check`

**Pull request closed without merging**